common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -179,11 +180,12 @@
                 <div class="fc-item__media-wrapper">
                     <div class="fc-item__image-container u-responsive-ratio fc-item__slideshow fc-item__slideshow--@imageElements.size">
                         @imageElements.headOption.map { imageElement =>
+                            @maxSlides = if(ActiveExperiments.isParticipating(NoMoreThanFiveSlides)) 5 else 10
                             @captionedImage(
                                 classes = Seq("responsive-img "),
                                 widths = item.mediaWidthsByBreakpoint,
                                 maybePath = Some(imageElement.url),
-                                maybeSrc = if(containerIndex == 0 && index < 9) Some(imageElement.url) else None,
+                                maybeSrc = if(containerIndex == 0 && index < maxSlides - 1) Some(imageElement.url) else None,
```

Comment:
What happens if the image is created but the src isn't set?

Reply:
Is there a chance we'll end up with a bunch of captions with no images, or a bunch of empty divs with nothing at all in? Perhaps we could slice or limit the array before the loop?

Reply:
Good point, I’ve refactored